### PR TITLE
[Bugfix] Wiki Modal imports correction + Refactor of Create New Page Modal

### PR DESCRIPTION
**Pull request by ajainuary**
**Description**
This PR fixes Modal Imports in the Wiki which were changed by #1793 
It also refactors the modal to create a new page and resolves the bug of modal not closing after creating a new page.

**Resolved Issues**
Import bug introduced by #1793 

**How to test?**
Use Page Settings to trigger the various Modal

### CODE REVIEW

File: frog/imports/client/Wiki/index.js [hunk 1]
```diff
@@ -687,6 +681,41 @@ class WikiComp extends React.Component<WikiCompPropsT, WikiCompStateT> {
     );
   };
 
+  createPageModal = () =>
```
Comment (RachitMalik12) [inline frog/imports/client/Wiki/index.js:684]: Would it possible for you to separate your work on PageSettings to another PR since this PR is focused on fixing the import bugs?

**Review by RachitMalik12 — COMMENTED** [review]
**Resolved**

File: frog/imports/client/Wiki/index.js (hunk 1 above)
Comment (ajainuary) [reply frog/imports/client/Wiki/index.js:684]: This is a part of refactor of modal

**Review by houshuang — CHANGES_REQUESTED** [review]
Everything looks good except that when you create a new individual page, two "Undefined" instances appear. I verified that this does not happen on develop.

Screenshot 2019-07-19 at 17 32 54
